minio/resource_minio_ilm_policy.go

```diff
@@ -208,3 +231,12 @@ func parseILMExpiration(s string) lifecycle.Expiration {
 
 	return lifecycle.Expiration{}
 }
+
+func parseILMNoncurrentVersionExpiration(s string) lifecycle.NoncurrentVersionExpiration {
+	var days int
+	if _, err := fmt.Sscanf(s, "%dd", &days); err == nil {
+		return lifecycle.NoncurrentVersionExpiration{NoncurrentDays: lifecycle.ExpirationDays(days)}
+	}
```

Comment:
I think we can simplify this if we use an integer instead of a string here

Reply:
i delete this part as it is not needed to parse an int.